### PR TITLE
Project 5: Views.

### PR DESCRIPTION
**Pull request by er-contreras**
- Implement the design from the sneak peek wireframes

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+<ul id='users_section'>
+  <% @users.each do |user| %>
+    <li class='users'>
+      <div class='users_pic'></div>
+      <div class='users_name'>
+          <h2>User <%= user.name %></h2>
+          <p>Number of posts: <%= user.posts_count %></p>
+      </div>
+    </li>
+  <% end %>
```
Comment (oxenprogrammer) [inline app/views/users/index.html.erb]: **Changes Required**
- [x] Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). Let us try and make these changes

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+    <div class='post'>
+      <h3>Post # <%= u.id %></h3>
+      <p> Text:<%= u.text %></p>
+      <div>
+        Comments: <%= u.comments_count %>
+        Likes: <%= u.likes_count %>
+      </div>
+    </div>
+    <% end %>
+  </ul>
```
Comment (oxenprogrammer) [inline app/views/users/show.html.erb:24]: **Changes Required**
- [x] Let us add a **See all Post** links that opens `user's post index` as described in the requirements.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @eri8-9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] Please add all the `necessary steps` to setup and run your project on a local machine.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @eri8-9, 

Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**